package-lock.json

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
Please delete this file, as we only need it in the frontend folder (already exists there). Thanks!